### ISSUE TITLE
websocket cannot get binary message

### ISSUE THREAD

**Issue by bianbian-org** [slact/nchan#459]
pub:
```
>> POST /pub/... HTTP/1.1
>> Content-Type: application/octet-stream
...
<< "HTTP/1.1 201 Created[\r][\n]"
<< "Server: nginx/1.15.0[\r][\n]"
...
<< "queued messages: 16[\r][\n]"
```

web:
ERROR TypeError: evt.data.match is not a function
    at WSWrapper.eval (NchanSubscriber.js:631)
js:
```
        l.onmessage = ughbind(function(evt) {
          var m = evt.data.match(/^id: (.*)\n(content-type: (.*)\n)?\n/m);
          this.emit("message", evt.data.substr(m[0].length), {"id": m[1], "content-type": m[3]});
        }, this);
```
when post binary message, evt.data is blob

**Comment by bianbian-org** [comment]
using raw WebSocket without nchan subprotocol is ok, so the problem maybe subprotocol or NchanSubscriber.js?

**Comment by slact** [comment]
This is fixed in the latest version of Nchan.js in https://github.com/slact/nchan.js/commit/e7e74e0143cd1623aa55bf97e6df3d6b36194838

**Comment by slact** [comment]
On second look, this fix was in master, but had not been published to NPM. I've now updated  and published  the Nchan NPM package version 1.0.10 that includes this fix.
